Web/starter_project_iconics/pages/profile/index.tsx

```diff
@@ -0,0 +1,35 @@
+import BlogCard from '@/components/common/blogCard'
+import { card1 } from '@/store/mockData/data'
```

Comment:
might be a better idea to rename card1
idk what card1 means as opposed to card2? is there even card2?